packages/ui-extensions/src/surfaces/customer-account/targets.ts

```diff
@@ -12,7 +21,12 @@ import type {RenderExtension} from './extension';
  * The input arguments and the output type are different
  * for each extension target.
  */
-export interface ExtensionTargets {
+export interface ExtensionTargets
+  extends OrderStatusExtensionTargets,
+    CustomerAccountExtensionTargets {}
```

Comment:
Can we have a proper union/intersection here ? I don't think we would ever have a member live in the interface's body ?